### ISSUE TITLE
Scroll bar has a habit of sticking

### ISSUE THREAD

**Issue by SebVen** [rstudio/rstudio#4691]
When scrolling by dragging the scroll bar in any RStudio window, the scroll bar often has a habit of 'sticking' causing the script or page to accidentally jump to different sections. This happens when while the scroll bar is dragged, the mouse cursor leaves the window (e.g. the cursor enters the Windows task bar, desktop or a second screen). When left-click is released, the scrollbar is still somehow 'selected'. Upon return of the cursor to the scroll bar (accidental or otherwise), the script/page jumps.

This unwanted jumping to different sections of the script is an annoyance as it requires returning to the correct section each time it happens. Not perhaps a high-priority bug, but it would be great if it could be fixed. It is not a phenomenon I've seen with other software.

RStudio edition: Desktop
RStudio ver: 1.2.1502
R ver: 3.5.2
OS: Windows 10 build 17763

**Comment by ronblum** [comment]
These are probably duplicates which we're not consistently able to reproduce.
- https://github.com/rstudio/rstudio/issues/4529
- https://github.com/rstudio/rstudio/issues/5145

**Comment by SebVen** [comment]
They may be possibly related to the same cause, but those two cases are slightly different from the issue I've encountered.

**Comment by kevinushey** [comment]
I don't think this meets the bar for v1.2-patch -- it will likely be too challenging to fix for a patch release.

**Comment by dcldmartin** [comment]
+1 to this. Happens fairly reliably to me after using RStudio for an extended period of time. Reverts to normal behavior when restarted.

**Comment by nesat** [comment]
I am a mac user and have the same problem

macOS Catalina 10.15.3
RStudio 1.2.5033
R 3.4.1

**Comment by brettmelbourne** [comment]
+1. Rstudio 1.2.5033. Windows 10. Exactly as original issue describes.
Open code in source editor.
Show new window.
Left click on scroll bar (the short, darker grey section) to scroll but keep holding mouse button and release outside the scroll bar to left or right. This is easy to do accidentally.
The scroll bar is now in a cursor sensitive mode: moving the mouse across the scrolling field will cause the scroll bar to jump to the position where the mouse crosses the window.
Left clicking on the scroll bar and carefully releasing the mouse button within the scroll bar will reset it to normal.
Basically you have to take great care to release the mouse within the scroll bar or you get this jumping behavior.

**Comment by Filippone2000** [comment]
I have the same problem:

RStudio edition: Desktop
RStudio ver: 1.4.1010
R ver: 4.0.3
OS: Windows 10 build 1909

**Comment by ronblum** [comment]
Closing this as fixed by Electron. If anyone using builds from [the current release (2022.11.0)](https://dailies.rstudio.com/rstudio/elsbeth-geranium/electron/macos/) continues to run into this issue, please let us know!

**Comment by guycox146** [comment]
+1 exactly as described - very annoying, but not with Electron, happens in all windows on Windows 10 Dell pc
